setup.py

```diff
@@ -68,11 +68,9 @@
     ],
     extras_require={
         'rllib': [
-            'GPUtil',
             'GPy',
```

Comment:
Sorry, and please revert this one too. If you wanted to add this or the other GPU dep to the Dockerfile, please do.